build.assets/build-fido2.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
+#
+# Builds libfido2 and dependencies, caching the resulting binaries in the local
+# filesystem.
+#
+# Run `build-fido2.sh build` to build libfido2 and its dependencies, at the
+# versions specified in the script.
+# Run `build-fido2.sh pkg_config_path` to print the path to the prior-built
+# libfido2-static.pc file.
+#
+# Written mainly for macOS builders.
```

Comment:
Should the filename of the script reflect that this is very much only for macOS?

Reply:
Sure. There's almost nothing macOS specific (there is `-mmacosx-version-min`, which I'm now convinced has almost no effect here), but since it lacks an udev alternative I think it's fair to make a stronger macOS mention.